### DISCUSSION TITLE
ClaimTooSmallForActiveBlocks

### DISCUSSION THREAD

**Discussion by SrBedrock** [GriefPrevention/GriefPrevention#652] [Ideas]
Where can I configure this part of the plugin?

ClaimTooSmallForActiveBlocks
TooManyActiveBlocksInClaim

**Comment by RoboMWM** [comment]
what

**Comment by RoboMWM** [comment]
are you asking to configure the messages?

Or the setting it refers to?

**Comment by SrBedrock** [comment]
I would like to know if it is possible to configure the amount of active blocks in the claim, not just the messages.

**Comment by RoboMWM** [comment]
Oh, this relates to hoppers in creative claims. Hmm, not sure if there's a config setting.

**Comment by SrBedrock** [comment]
It would be a nice addition, accumulation of active blocks in the protections causes a large loss of TPS in version 1.14.4.
